listenbrainz/webserver/static/js/src/types.d.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
Should we just disable this everywhere? I'd like to keep the convention of using camelcase everywhere, but there are some places where it's just not possible (stuff coming from python etc), so maybe we should just get rid of this.

Reply:
I think it's fine to keep it for the jsx pages. I don't mind changing that file-wide disable into specific blocks disables for the items we know for sure like spotify API and python types

Reply:
Sounds good to me!

Reply:
Sounds good to me!